src/plugins/mdn/mdnPlugin.js

```diff
@@ -70,6 +70,18 @@ async function fixLanguage(origRes, lastRedirect) {
   return res;
 }
 
+async function fixRedirect(res) {
+  const $ = cheerio.load(res.text);
+  const script = $('script').get()[0].children[0].data;
+  const reg = /(window.location.replace\(\'\/l\/\?kh=-1&uddg=)(.*)(\'\))/;
+  const redirect = decodeURIComponent(script.match(reg)[2]);
```

Comment:
I'd check if there's a match or not and just return `res` if not.

Also I don't think `decodeURIComponent` is appropriate here, so can remove that.  We should verify the URL by passing it to `new URL`, and checking that the protocol is http or https, and the domain ends with '.mozilla.org'.